jwst/lib/exposure_types.py

```diff
@@ -0,0 +1,23 @@
+"""
+This module contains lists of modes grouped in different ways
+"""
+from ..associations.lib.dms_base import (ACQ_EXP_TYPES, IMAGE2_SCIENCE_EXP_TYPES,
+                                         IMAGE2_NONSCIENCE_EXP_TYPES,
+                                         SPEC2_SCIENCE_EXP_TYPES)
+
+IMAGING_TYPES = set(tuple(ACQ_EXP_TYPES) + tuple(IMAGE2_SCIENCE_EXP_TYPES)
+                    + tuple(IMAGE2_NONSCIENCE_EXP_TYPES) +
+                    ('fgs_image', 'fgs_focus'))
+
+SPEC_TYPES = SPEC2_SCIENCE_EXP_TYPES
+
+# JWST Exposures that are Fine Guidance exposures that actually
```

Comment:
A really minor nit, but "Fine Guidance" is actually the name of one of the FGS guiding modes (ID, Acq, Track, Fine Guide), so a more appropriate wording for any type of guiding mode exposures would be "FGS guide star exposures". And technically they aren't even "exposures"; they're a continuous data stream that lasts as long as a given type of guiding activity takes to complete (e.g. Track and Fine Guide data could span time covering many exposures), but that's an even more minor nit that we won't worry about here.